src/App.js

```diff
@@ -0,0 +1,235 @@
+import React, {useEffect, useState} from 'react';
+import './App.css';
+import AudioKey from './components/audioKey'
+import BackgroundSounds from './components/BackgroundSounds'
+
+const keys = 'abcdefghijklmnopqrstuvwxyz'.split('');
+
+const colours = [];
+
+for(let i = 0; i < 26; i++){
+  colours.push(`rgb(${Math.round(Math.random() * 255)},${Math.round(Math.random() * 255)},${Math.round(Math.random() * 255)})`)
+}
+
+const glassAudioFiles = [
+  "A501.wav",
+  "A503.wav",
+  "A505.wav",
+  "A507.wav",
+  "As4-B401.wav",
+  "As4-B403.wav",
+  "As4-B405.wav",
+  "As502.wav",
+  "As504.wav",
+  "As506.wav",
+  "B401.wav",
+  "B403.wav",
+  "B405.wav",
+  "A502.wav",
+  "A504.wav",
+  "A506.wav",
+  "A508.wav",
+  "As4-B402.wav",
+  "As4-B404.wav",
+  "As501.wav",
+  "As503.wav",
+  "As505.wav",
+  "As507.wav",
+  "B402.wav",
+  "B404.wav",
+  "B406.wav",
+
+  // "E601.wav",
```

Comment:
probably a good idea to remove this many commented out lines from final deployment, for code cleanliness